src/app/test/db/mongo/models/garden_model_test.py

```diff
@@ -0,0 +1,311 @@
+# -*- coding: utf-8 -*-
+import copy
+from contextlib import nullcontext as does_not_raise
+
+import pytest
+from mongoengine import NotUniqueError, connect
+from mongoengine.errors import ValidationError
+
+from beer_garden.db.mongo.models import Garden as MongoGarden
```

Comment:
I've actually already done the opposite of this in a lot of cases, which is alias any instances of Brewtils models to something like `BrewtilsGarden` so that the beer garden models can just go by their name, like `Garden`.  I vote that you keep it consistent here and just import the models as `Garden` and `System`.

I didn't really look at these tests in my original review, so I apologize for the delayed feedback.